### PR TITLE
[DEVHUB-1368]: Home Page Markup & Styles

### PR DESCRIPTION
**Pull request by justinmckenzie**
Scope:
- Adds remaining markup and styles to complete Home Page
- Adds base level unit tests for all existing components (Unit tests will be expanded during/after integration)
- Adds CSS variables

### CODE REVIEW

File: src/components/featured-card/styles.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  flexDirection: ['column', 'row', null, null],
+  alignItems: 'center',
+  paddingTop: ['inc50', 'inc70', null, 'inc80'],
+  paddingBottom: ['inc70', null, null, 'inc80'],
+  paddingLeft: ['inc70', null, null, 'inc100'],
+  paddingRight: ['inc70', null, null, 'inc100'],
+  marginRight: [null, null, null, 'inc40'],
+  marginBottom: ['inc40', null, null, 0],
+  boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.1)',
+  border: '1px solid #E7EEEC',
```
Comment (mcolella14) [inline src/components/featured-card/styles.ts:18]: These could be theme values as well.

File: src/styles/variables.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// Colors
+export const COLOR_WHITE = '#FFFFFF';
+export const COLOR_DARK_PRIMARY = '#001E2B';
+export const COLOR_DARK_SECONDARY = '#023430';
```
Comment (mcolella14) [inline src/styles/variables.ts:4]: So these are all theme values, and I would lean towards using those instead. In general, If you go to [this file](https://github.com/10gen/flora/blob/main/tokens.js) and just search for the color, you can find the corresponding theme value.

**Review by mcolella14 — APPROVED** [review]
Nothing blocking, everything looks great! Just some nitpicky things about theme stuff.

File: src/styles/variables.ts (hunk 2 above)
Comment (justinmckenzie) [reply src/styles/variables.ts:4]: I totally agree, I wanted to use directly from theme values too but I just couldn't find anywhere to reference them lmao. I will change these to theme values.

**Pull request merged**